### ISSUE TITLE
Warning "Illegal command \ifile found as part of a \c command" in \copydoc, but not original documentation

### ISSUE THREAD

**Issue by StarFire75** [doxygen/doxygen#9584]
I get the warning "Illegal command \ifile found as part of a \c command" for the following documentation:

```
        /** \brief Validate pointer
          *
          * Validate a parameter of type pointer. It is checked if the
          * pointer is \c nullptr and if so the exception is thrown
          *
          * \param p_Pointer Pointer to validate
          * \param p_Message Exception message if invalid
          *
          * \exception std::invalid_argument if the pointer is \c nullptr
          */
        static void ValidateParameter(void* p_Pointer, std::string const& p_Message);
        //! \copydoc ValidateParameter(void* p_Pointer, std::string const& p_Message)
        static void ValidateParameter(const void* p_Pointer, std::string const& p_Message);

```
Marked for the warning is the last line with actual documentation ("\exception...").

In the generated documentation the first method is correctly rendered; in the second one however, I get the following line instead the exception line:
std::invalid_argument if the pointer is `nullptr\ifile  "J:/QUARTIS/_dev/src/System.C++/Validator.h"`

This happens in version 1.9.5, but not in 1.9.4

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #9586

**Comment by albert-github** [comment]
Code has been integrated in master on GitHub (please don't close the issue as this will be done at the moment of an official release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.6.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
